src/application.coffee

```diff
@@ -150,8 +150,12 @@ fetch = (app, setDeviceUpdateState = true) ->
 		device.updateState(status: 'Downloading', download_progress: 0)
 
 		Promise.try ->
-			JSON.parse(app.env)
-		.then (env) ->
+			# In dependent apps env is replaced with config
```

Comment:
This will receive apps processed in `parseEnvAndFormatRemoteApps`, so for non-dependent apps it will still be `env` (until we actually start using `config`)